docs/src/man/getting_started.md

```diff
@@ -128,14 +128,22 @@ dt = DataTable(A = 1:4, B = randn(4))
 colwise(c->cumsum(dropnull(c)), dt)
 ```
 
+
+## IO
+
+For reading and writing tabular data, load the [CSV.jl](https://github.com/JuliaData/CSV.jl) package with
```

Comment:
Should also say and show how to install it. Anyway `Pkg.add` has no effect if the package is already installed.

Also, "tabular" data is too broad: "tabular data from CSV and other delimited text files" would be more accurate. At some point we should add instructions about DataStreams and other formats.